src/templates/EventList.html

```diff
@@ -1,4 +1,4 @@
-<div id="event-list" :class="['event-list', { '-active': view === 'list'}]">
+<div id="event-list" class="event-list" :class="['event-list', { '-active': view === 'list'}]">
```

Comment:
Note to self: per this comment https://github.com/aclu-people-power/map/pull/86#discussion_r111672073, I don't believe `event-list` in the `:class` bound array is doing anything, I'll chop that out.